app/src/main/java/com/example/gitstalker/MainActivity.java

```diff
@@ -50,7 +51,7 @@ public void onStart() {
         super.onStart();
         Branch branch = Branch.getInstance();
 
-        // Branch init
+        // Branch init callback
```

Comment:
@sanasimran you are using branch.initSession() which is pretty old. This would solve the use case however you can also refer to [initialize Branch](https://help.branch.io/developers-hub/docs/android-basic-integration#initialize-branch) section for an updated callback.